sycl/test/xocc_tests/sdaccel_ports/vision/edge_detection/edge_detection_with_webcam.cpp

```diff
@@ -78,7 +78,7 @@ int main(int argc, char* argv[]) {
             for(size_t k = 0; k < 3; ++k) {
               for(size_t l = 0; l < 3; ++l) {
```

Comment:
It would be nice to have these 2 loops correctly formatted in the same way as the previous ones